test/docgen/view/__snapshots__/class.test.ts.snap

```diff
@@ -87,6 +87,96 @@ Grant access to retrieve an authorization token.
 ---
 
 
+"
+`;
+
+exports[`go snapshot 1`] = `
+Object {
+  "constants": Array [],
+  "displayName": "AuthorizationToken",
+  "docs": Object {
+    "links": Array [
+      "https://docs.aws.amazon.com/AmazonECR/latest/userguide/registry_auth.html",
+    ],
+    "summary": "Authorization token to access private ECR repositories in the current environment via Docker CLI.",
+  },
+  "fqn": "github.com/aws/aws-cdk-go/awscdkawsecr.AuthorizationToken",
+  "id": "@aws-cdk/aws-ecr.AuthorizationToken",
+  "initializer": undefined,
+  "instanceMethods": Array [],
+  "interfaces": Array [],
+  "properties": Array [],
+  "staticMethods": Array [
+    Object {
+      "displayName": "GrantRead",
+      "docs": Object {
+        "summary": "Grant access to retrieve an authorization token.",
+      },
+      "fqn": "github.com/aws/aws-cdk-go/awscdkawsecr.AuthorizationToken.GrantRead",
+      "id": "@aws-cdk/aws-ecr.AuthorizationToken.grantRead",
+      "parameters": Array [
+        Object {
+          "default": undefined,
+          "displayName": "grantee",
+          "docs": Object {},
+          "fqn": "github.com/aws/aws-cdk-go/awscdkawsecr.AuthorizationToken.GrantRead.parameter.grantee",
+          "id": "@aws-cdk/aws-ecr.AuthorizationToken.grantRead.parameter.grantee",
+          "optional": undefined,
+          "type": Object {
+            "formattingPattern": "%",
+            "types": Array [
+              Object {
+                "displayName": "IGrantable",
+                "fqn": "undefined/awscdkawsiam.IGrantable",
+                "id": "@aws-cdk/aws-iam.IGrantable",
+                "packageName": "@aws-cdk/aws-iam",
+                "packageVersion": "99.99.99",
+              },
+            ],
+          },
+        },
+      ],
+      "usage": "import \\"github.com/aws/aws-cdk-go/awscdkawsecr\\"
```

Comment:
This is correct actually as the fixtures are configured. This doesn't match the v2 module names because the moduleNames are dummy values manually set on the fixtures .jsii assembly and the packageNames are auto generated by jsii-pacmak, whereas in V2 packageNames are generated from submodule configuration + file structure.

Reply:
Does this mean the imports are always going to be incorrect for v1 docs? On Construct Hub for example should we not show Go as an available language for any AWS CDK v1 libraries?

Reply:
If the fixtures are outdated it's probably a good opportunity to update them

Reply:
So I'm updating the fixtures with the missing `moduleName` values to avoid `undefined`s, I don't think it's worth blowing away all of our snapshots to update these currently. In a separate PR maybe otherwise it's a lot of noise?

Go won't show for v1 modules in construct-hub because they don't have Go enabled. Just like other languages that a module doesn't support.

Reply:
Cool, yeah a separate PR is fine. 👍